backend/haal-centraal-auth/build.gradle.kts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dockerCompose {
+    setProjectName("HaalCentraalAuth")
+    isRequiredBy(project.tasks.integrationTesting)
+
+    val valtimoVersion: String by project
+
+    dependencies {
+        implementation(platform("com.ritense.valtimo:valtimo-dependency-versions:$valtimoVersion"))
+        implementation("com.ritense.valtimo:valtimo-gzac-dependencies")
+        implementation("com.ritense.valtimo:wordpress-mail:$valtimoVersion")
+        implementation("com.ritense.valtimo:besluit:$valtimoVersion")
+        implementation("com.ritense.valtimo:objects-api:$valtimoVersion")
+        implementation("com.ritense.valtimo:openzaak-resource:$valtimoVersion")
+        implementation("com.ritense.valtimo:openzaak:$valtimoVersion")
+        implementation("com.ritense.valtimo:smartdocuments:$valtimoVersion")
+        implementation("com.ritense.valtimo:wordpress-mail:$valtimoVersion")
+        implementation("com.ritense.valtimoplugins:freemarker:6.0.0")
+        implementation("com.ritense.valtimoplugins:object-management:0.2.0")
```

Comment:
Im unable to startup the application. I think its because of these dependencies. They are not needed